search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -331,6 +331,18 @@
                                     not-empty)]
      (str (string/join ". " citation-details) "."))))
 
+(defn- nil-if-not-provided
+  "nil if not provided."
+  [s]
+  (when-not (= umm-spec-util/not-provided s)
+    s))
+
+(defn graphic-preview-type
```

Comment:
If we don't have a graphic preview type, will it still render on data.gov? e.g. Are they figuring out the type based on the extension in the URL provided if they can?

Reply:
Yes it will display regardless of what graphic-preview-type is set to.